### ISSUE TITLE
Support VoiceOver

### ISSUE THREAD

**Issue by jake-low** [mapbox/mapbox-maps-ios#1024]
## What

Add basic support for VoiceOver (iOS's built-in screen reader) to map elements, in order to reach parity with Mapbox iOS SDK v6.

## Why

VoiceOver allows people with limited vision to use iOS apps. It can read aloud text that's shown on the screen, and describe user interface controls like buttons and sliders when a user touches them.

Without VoiceOver support, app developers can't upgrade to Mapbox SDK v10 without breaking their apps for some of their users.

## Details

I'm currently porting an app built on the legacy Mapbox iOS SDK (v5.x) to Mapbox SDK v10. During this process I discovered that my new Mapbox SDK v10-based map is not accessible to VoiceOver users.

Mapbox's legacy iOS SDK (versions 3.7.0 through 6.x) had impressive support for VoiceOver. See [this blog post](https://blog.mapbox.com/accessible-maps-for-blindness-or-visual-impairment-on-ios-5c7911c349c4) for some example videos of how it works. You can also build and install the [legacy examples app](https://github.com/mapbox/ios-sdk-examples/) and then [enable VoiceOver](https://support.apple.com/guide/iphone/turn-on-and-practice-voiceover-iph3e2e415f/ios) to try it out for yourself.

Mapbox SDK v10 appears to have no support for VoiceOver. Map elements (like POIs and roads) are not accessible. The only accessible elements on-screen appear to be the compass button (upper right) and the "more info" button (lower right). You can verify this by building the [examples app in this repository](https://github.com/mapbox/mapbox-maps-ios/tree/main/Apps/Examples) and trying it out with VoiceOver enabled.

If I'm wrong, and SDK v10 does support VoiceOver, perhaps you could update the example code to demonstrate this? And if not, I'm hopeful that VoiceOver support is on the roadmap for a near-future 10.x release.

Best regards,
Jake

**Comment by jojost1** [comment]
Any updates on this? Wondering if this is something that we can expect to return to Mapbox or not.

**Comment by aanthone** [comment]
Hello team, similar to @jojost1, I was wondering if there is an update on this. Ancestry is asking does Mapbox have a document that talks about mobile accessibility for the maps, specifically for iOS voiceover? They would like more control over what is being displayed to voiceover users as they address accessibility concerns for their user demographics. 
cc @jake-low
